src/lib/containers/table/SynapseTable.tsx

```diff
@@ -411,22 +407,6 @@ export default class SynapseTable extends React.Component<
               queryBundleRequest={queryRequest}
             />
           )}
-          {!showFacetFilter &&
-            unitDescription &&
-            !isGroupByInSql(queryRequest.query.sql) && (
-              <div
-                className={`SRC-centerContent text-left`}
-                style={{ minHeight: '20px' }}
-              >
-                <TotalQueryResults
-                  style={{ fontSize: 15 }}
-                  frontText={'Showing'}
-                  applyChanges={(newFacets: FacetColumnRequest[]) =>
-                    this.applyChangesFromQueryFilter(newFacets)
-                  }
-                />
-              </div>
-            )}
```

Comment:
I believe with these changes, TotalQueryResults can no longer be shown in a StandaloneQueryWrapper if no TopLevelControls is shown. Not sure if this is unintended, and not sure if it's a big deal--depends on how StandaloneQueryWrapper is typically configured in portals/SWC.